tests/coord.test.js

```diff
@@ -0,0 +1,46 @@
+const Coord = require('../coord.js');
+
+test('initialize Coord', () => {
+    // x and y init to negative values, so 0 is different enough
+    let coord = new Coord(0, 0);
+
+    expect(coord.x).toBe(0);
+    expect(coord.y).toBe(0);
+});
+
+test('modify Coord', () => {
+    let coord = new Coord(0, 0);
+
+    coord.x = 10;
+    coord.y = 15;
+
+    expect(coord.x).toBe(10);
+    expect(coord.y).toBe(15);
+
+    coord.setLocation(20, 25);
```

Comment:
You should split everything above this into one test and everything below into another. You want your tests to be atomic, and the top is testing one thing and the bottom is testing something else.